plans/git_reference.py

```diff
@@ -4,11 +4,23 @@
 from pathlib import Path
 
 import fmf
+import requests
 
+# Set discover of specfile tests to a fixed commit
 tree_root = Path.cwd().absolute()
-node = fmf.Tree(tree_root).find("/plans")
-with node as data:
+tree = fmf.Tree(tree_root)
+main_node = tree.find("/plans")
+with main_node as data:
     data["discover"]["url"] = "https://github.com/packit/specfile.git"
     data["discover"]["ref"] = (
         subprocess.check_output(["git", "rev-parse", "HEAD"]).decode().strip()
     )
+
+
+# Set discover of packit integration tests to the current main
+packit_node = tree.find("/plans/packit-integration")
+r = requests.get("https://api.github.com/repos/packit/packit/commits/main")
+if not r.ok:
+    raise SystemExit(1)
+with packit_node as data:
+    data["discover"]["ref"] = r.json()["sha"]
```

Comment:
Since we are already calling git above, wouldn't it be better to call
`git ls-remote https://github.com/packit/packit main` to get the SHA?